admin/assets/stylesheets/application.scss

```diff
@@ -333,3 +333,25 @@ table.spacious-two-column {
   margin-top: 4.4em;
 }
 
+//not sure where to place these yet
```

Comment:
if unsure about placement speak to Jon

Reply:
May be best to create a new partial just for the sorting